README.md

```diff
@@ -63,20 +177,13 @@ let client = Client(spaceId: "cfexampleapi",
                     clientConfiguration: clientConfiguration)
 ```
 
-The `client` should be configured. Let's use it to fetch an entry.
+### Authorization
 
-```swift
-client.fetchEntry(id: "nyancat") { (result: Result<Entry>) in
-    switch result {
-        case .success(entry):
-            print(entry)
-        case .error(let error):
-            print("Error \(error)!")
-    }
-}
-```
+Grab credentials for your Contentful space by [navigating to the "APIs" section of the Contentful Web App](https://app.contentful.com/deeplink?link=api).
+If you don't have access tokens for your app, create a new set for the Delivery and Preview APIs.
+Next, pass the id of your space and delivery access token into the initializer like so:
 
-## `EntryDecodable`
+### `EntryDecodable`
```

Comment:
I'd extend that headline with a description what it actually is:

### Map Contentful Content Types to Swift classes via `EntryDecodable`